src/onnx/parse_split.cpp

```diff
@@ -55,12 +55,16 @@ struct parse_split : op_parser<parse_split>
         {
             literal s = parser.parse_value(info.attributes.at("split"));
             s.visit([&](auto v) { vec_splits.assign(v.begin(), v.end()); });
-
-            if(std::accumulate(vec_splits.begin(), vec_splits.end(), int64_t(0)) !=
-               static_cast<int64_t>(lens[tuned_axis]))
+        }
+        else if(args.size() == 2)
+        {
+            if(not args[1]->can_eval())
             {
-                MIGRAPHX_THROW("PARSE_SPLIT: sum of split attribute unequal to dim size of axis!");
+                MIGRAPHX_THROW("PARSE_SPLIT: invalid split input arg");
             }
+
+            auto s = args[1]->eval();
```

Comment:
Use `check_arg_empty(s, "Split: dynamic shape is not supported")`. Like what we do in `parse_reshape`.

Reply:
done